### ISSUE TITLE
[t-tagInput] 在有@submit的表单中回车会触发表单提交

### ISSUE THREAD

**Issue by ccfjzf2022** [Tencent/tdesign-vue#963]
### tdesign-vue 版本

0.41.7

### 重现链接

https://codesandbox.io/s/tdesign-vue-demo-forked-bio1p9?file=/src/demo.vue

### 重现步骤

```vue
<template>
  <div style="width: 350px; padding: 50px">
    <t-form
      ref="form"
      :data="formData"
      @reset="onReset"
      @submit="onSubmit"
      :colon="true"
      :labelWidth="0"
    >
      <t-form-item name="account">
        <t-tagInput />
      </t-form-item>
      <t-form-item style="margin-left: 100px">
        <t-button theme="primary" type="submit" block>登录</t-button>
      </t-form-item>
    </t-form>
  </div>
</template>
<script>
export default {
  data() {
    return {
      formData: {},
    };
  },
  methods: {
    onReset() {
      this.$message.success("重置成功");
    },
    onSubmit({ validateResult, firstError }) {
      if (validateResult === true) {
        this.$message.success("提交成功");
      } else {
        console.log("Errors: ", validateResult);
        this.$message.warning(firstError);
      }
    },
  },
};
</script>
```

### 实际结果

![image](https://user-images.githubusercontent.com/77626468/172370175-69a8aae1-259f-4481-b224-ed4d62252538.png)

**Comment by chaishi** [comment]
resolved in https://github.com/Tencent/tdesign-vue/pull/983

待发布

**Comment by uyarn** [comment]
fixed in 0.42.1
